### PR TITLE
[release] merging crate-release-update into crate

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @GuillaumeGomez @EPashkin

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,8 +12,11 @@ addons:
     - libgtk-3-dev
 script:
   - rustc --version
+  - if [ "$TRAVIS_RUST_VERSION" == "nightly" ] && [ "$GTK" == "3.4" ]; then
+    make regen_check;
+    fi
   - mkdir .cargo
   - echo 'paths = ["."]' > .cargo/config
-  - git clone -q --depth 50 -b pending https://github.com/rust-gnome/examples _examples
+  - git clone -q --depth 50 -b pending https://github.com/gtk-rs/examples _examples
```
[the remark this answers is not in the document]
Comment (EPashkin) [reply .travis.yml]: In this case IMHO you better add it (not necessary before publish) `-b master`

**Comment by EPashkin** [comment]
Sys crate branch has tag 0.8.0 while all in it 0.7.0

**Comment by EPashkin** [comment]
Same problem with cairo it has 0.7.0 (as sys) not as main 0.5.0

**Comment by EPashkin** [comment]
and with sourceview

**Comment by GuillaumeGomez** [comment]
Hum... I don't really know how to handle multiple crate versions in a single repository... I'll try to think something about (and eventually fix version numbers).

**Comment by EPashkin** [comment]
Oh, 0.8.0 go from `pangocairo-sys` I missed that, then it really same problem.

**Comment by EPashkin** [comment]
As variant we can add prefix to all version tags (maybe only for these repos),
"atk-sys0.7.0", "gtk-sys0.7.0", "pangocairo-sys0.8.0"
"cairo0.5.0", "cairo-sys0.7.0"

**Comment by GuillaumeGomez** [comment]
not sure if better since these tags will all point to the same commit... I really don't know what we should do in here.

**Comment by EPashkin** [comment]
IMHO It normal that it point to same commit currently, but if we have new version for Ex. "cairo" without sys, we see that it only  changes.

**Comment by GuillaumeGomez** [comment]
I'll think about how to solve this. Meanwhile I'll open a PR on the release repository so we don't forget about it.

**Pull request merged**